plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationContext.java

```diff
@@ -30,6 +30,8 @@
 
     private transient final ConfiguratorRegistry registry;
 
+    private transient String mode;
```

Comment:
is this still required?

Reply:
IMO it would be best to call `describeStructure` only when we need to generate the schema so keeping the mode and `isJsonSchema` in the general `_describe` function serves the purpose correctly